proposals/acp/wac-acp-diff-story.md

```diff
@@ -0,0 +1,127 @@
+# ACP Compared to Web Access Control
+
+Audience: those familiar with WAC or those who learn from comparisons
+
```

Comment:
Indicate that this is for the purpose of discussion/exercise -- as other things are moving in parallel - and not to be taken as authoritative definitions of the mechanisms.